### PR TITLE
Ollama: change testing workflow

### PR DESCRIPTION
**Pull request by anakin87**
- testing Ollama using Docker started failing: https://github.com/deepset-ai/haystack-core-integrations/actions/runs/8164727809
- the integrations works correctly (locally tested)
- Github services are sometimes unreliable and Ollama can be installed without Docker

So I am dropping the installation via Docker and I'm installing it following [the official instructions](https://github.com/ollama/ollama?tab=readme-ov-file#linux).

### CODE REVIEW

File: integrations/ollama/pyproject.toml [hunk 1]
```diff
@@ -157,20 +157,17 @@ ban-relative-imports = "parents"
...
 [tool.coverage.report]
+omit = ["*/tests/*", "*/__init__.py"]
+show_missing=true
 exclude_lines = [
-    "no cov",
-    "if __name__ == .__main__.:",
-    "if TYPE_CHECKING:",
+  "no cov",
+  "if __name__ == .__main__.:",
+  "if TYPE_CHECKING:",
```
Comment (anakin87) [inline integrations/ollama/pyproject.toml:170]: unrelated to this PR.
I just want the right coverage to be shown.

File: .github/workflows/ollama.yml [hunk 2]
```diff
@@ -33,16 +33,18 @@ jobs:
...
-          - 11434:11434
         
     steps:   
       - uses: actions/checkout@v4
 
+      - name: Install Ollama and pull the required models
+        run: |
+          curl -fsSL https://ollama.com/install.sh | sh
+          ollama serve &
+          sleep 10
```
Comment (wochinge) [inline .github/workflows/ollama.yml:44]: is there a health check for serve which we can use to see if the server is up? That might be a bit more robust than the hard coded sleep. Up to you though

**Review by wochinge — APPROVED** [review]
One tiny question. Otherwise looking good.

File: .github/workflows/ollama.yml (hunk 2 above)
Comment (anakin87) [reply .github/workflows/ollama.yml]: good idea. Done!

**Pull request merged**